src/main/java/com/faforever/api/map/MapService.java

```diff
@@ -343,11 +342,9 @@ private void validateScenarioLua(MapLuaAccessor mapLua, MapNameBuilder mapNameBu
     Optional<Map> existingMapOptional = mapRepository.findOneByDisplayName(displayName);
     existingMapOptional
       .ifPresent(existingMap -> {
-        Optional.ofNullable(existingMap.getAuthor())
-          .filter(existingMapAuthor -> !Objects.equals(existingMapAuthor, author))
-          .ifPresent(existingMapAuthor -> {
-            throw ApiException.of(ErrorCode.MAP_NOT_ORIGINAL_AUTHOR, existingMap.getDisplayName());
-          });
+        final Player existingMapAuthor = existingMap.getAuthor();
+        if (existingMapAuthor == null || !existingMapAuthor.equals(author))
```

Comment:
I think it was code style convention to always use brackets for if/else even for one liners

Reply:
yeah not sure how this has happened :), I don't like bracketless ifs too